cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,24 @@
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+import { CheckoutPageObject } from '../support/pages/checkout.pageObject';
 
-  });
+describe('Demoblaze app', () => {
+  let customer;
 
-  it('', () => {
+  beforeEach(() => {
+    cy.task('generateCustomer').then((generateCustomer) => {
+      customer = generateCustomer;
+    });
+  });
 
+  it('should provide an ability to buy an itemm', () => {
+    const checkoutPage = new CheckoutPageObject();
+    checkoutPage.visit();
```

Comment:
```suggestion
    checkoutPage.visit();
```
you use different pages in the test, so in future is better to create separate page object for every page